crypto/models.go

```diff
@@ -32,6 +32,10 @@ const (
 	PS256 SignatureAlgorithm = "PS256"
 )
 
+func (kt KeyType) String() string {
```

Comment:
I'm more used to .ToString() (java brain here) but if this is the better Go convention we can stick with this

Reply:
yeah, I think it is more of a go thing:
- https://github.com/xeipuuv/gojsonschema/blob/master/result.go#L68
- https://github.com/lestrrat-go/jwx/blob/ec54ec431aa503ab1d5fc11a5c97a70daf2e0171/jwt/options_gen_test.go#L12